public/static/docs/command-reference/update.md

```diff
@@ -69,3 +77,43 @@ stable.
 > Note that `dvc update` updates the `rev_lock` field of the corresponding
 > [DVC-file](/doc/user-guide/dvc-file-format) (when there are changes to bring
 > in).
+
+## Example: Updating imported artifacts to a specified revision
+
+Let's import a data artifact from an older commit from our
+[get started example repo](https://github.com/iterative/example-get-started) at
+first:
+
+```dvc
+$ dvc import --rev 72e0f1 git@github.com:iterative/example-get-started model.pkl
```

Comment:
don't rely on commit hash - they change in that repo

Reply:
split snippet into multiple lines

Reply:
instead of a hash - may be tag is more or less enough - tags might change, but in most cases they are not ... not sure about this